examples/raphtory-example-lotr/src/main/scala/com/raphtory/examples/lotr/PulsarOutputRunner.scala

```diff
@@ -9,27 +12,38 @@ import com.raphtory.sinks.PulsarSink
 import com.raphtory.spouts.FileSpout
 import com.raphtory.utils.FileUtils
 
-object PulsarOutputRunner extends App {
-
-  val path = "/tmp/lotr.csv"
-  val url  = "https://raw.githubusercontent.com/Raphtory/Data/main/lotr.csv"
-
-  FileUtils.curlFile(path, url)
-
-  // Create Graph
-  val source  = FileSpout(path)
-  val builder = new LOTRGraphBuilder()
-  val graph   = Raphtory.load(spout = source, graphBuilder = builder)
-
-  // Run algorithms
-  graph
-    .at(30000)
-    .past()
-    .execute(EdgeList())
-    .writeTo(PulsarSink("EdgeList"))
-  graph
-    .range(20000, 30000, 10000)
-    .window(List(500, 1000, 10000), Alignment.END)
-    .execute(PageRank())
-    .writeTo(PulsarSink("PageRank"))
+object PulsarOutputRunner extends IOApp {
+
+  override def run(args: List[String]): IO[ExitCode] = {
+
+    val path = "/tmp/lotr.csv"
+    val url  = "https://raw.githubusercontent.com/Raphtory/Data/main/lotr.csv"
+
+    FileUtils.curlFile(path, url)
+
+    // Create Graph
+    val source  = FileSpout(path)
+    val builder = new LOTRGraphBuilder()
+    Raphtory.loadIO(spout = source, graphBuilder = builder).use { graph =>
```

Comment:
Can this be changed to load. We do need some examples of loadIO/streamIO, but outside the LOTR examples as that is the first entry point.